config/config.exs

```diff
@@ -37,6 +37,8 @@ config :livebook,
   within_iframe: false,
   allowed_uri_schemes: []
 
+config :aws_credentials, fail_if_unavailable: false
```

Comment:
I am contributing to a new aws_credentials version where this is no longer necessary. So:

```suggestion
# TODO: Remove this in aws_credentials 0.2.0
config :aws_credentials, fail_if_unavailable: false
```